src/resolvers/validateEnumMembership.ts

```diff
@@ -0,0 +1,17 @@
+import { UserInputError } from 'apollo-server'
+import { difference } from 'lodash'
```

Comment:
Nit (non-blocking): this is a tiny optimization but this import statement will add all of lodash's modules to the bundle. You could do this instead:

```ts
import difference from 'lodash/difference'
```

https://selleo.com/til/posts/0ac258eyv8-decrease-bundle-size-by-importing-lodash-correctly

Reply:
We're not building a bundle for server-side code so wouldn't this have no effect?

Reply:
Okay yeah I need some sleep 🤦

Reply:
😄 well-earned, thanks for the review!